user/nmap/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "nmap"
+pkgver = "7.95"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--with-openssl",
+    "--without-zenmap",
+    "--without-ndiff",
+    "--without-ncat",
+]
+configure_gen = []
```

Comment:
```suggestion
```
fix the regen instead

Reply:
Sorry for the newbie question/wall of text. `autoreconf` is failing currently because the included libssh2's `Makefile.am` is requiring the [`tests/` and `example/`](https://github.com/nmap/nmap/blob/667527c4b46abb9cceef7a8b30a1f8a7e5d04d49/libssh2/Makefile.am#L3) directories which don't exist in the nmap included libssh2. My first inclination is to stop `autoreconf` from configuring that subdirectory since nmap is using the host libssh2 anyway. The problem is I don't know how to go about doing this :/ Insofar as I can tell `autoreconf` only has the `--no-recursive` flag which isn't useful because some subdirectories should be configured. I have maybe a few ideas, but I they all seem like a bodge. 
1. Modify the root `configure.ac` file to remove the `AC_CONFIG_SUBDIRS` (via patch file?)
2. Modify the `libssh2/Makefile.am` file to remove the requirement for `tests/` and `example/`
3. Run `autoreconf` with `--no-recursive` and then have cbuild manually recurse into only the subdirectories that actually need to be configured.

Getting your thoughts on this would be greatly appreciated. 💜

P.S. Is it better to ask this kind of thing in the IRC/matrix?

Reply:
> P.S. Is it better to ask this kind of thing in the IRC/matrix?  

either place is alright :)  

> Modify the root configure.ac file to remove the AC_CONFIG_SUBDIRS (via patch file?)

yeah, a patch to remove the _subdirs call to libssh2 sounds ideal in this case